Vagrantfile.openstack

```diff
@@ -0,0 +1,111 @@
+# -*- mode: ruby -*-
```

Comment:
Do you mind integrating this into the default Vagrantfile?
We currently support Virtualbox, VMware and Libvort providers within the same file so this shouldn't be an issue.

At some point, I'm hoping to have this branch merged with master so we should keep that in mind :)